src/services/tx/txSender.ts

```diff
@@ -262,7 +263,14 @@ export const dispatchTxExecution = async (
         txDispatch(TxEvent.PROCESSED, { txId, receipt })
       }
     })
-    .catch((error) => {
+    .catch((err) => {
+      const error = err as EthersError
+
+      // Meaning the transaction was "repriced" or "replaced"
+      if (error.code === ErrorCode.TRANSACTION_REPLACED && error.reason !== 'cancelled') {
+        // TxEvent.SUCCESS is handled by txHistoryMiddleware
+        return
+      }
       txDispatch(TxEvent.FAILED, { txId, error: error as Error })
```

Comment:
We should abstract this and use it for the multisend dispatcher as well.

Reply:
I've created a `isReprice` util